nautobot/virtualization/filters.py

```diff
@@ -202,21 +303,74 @@ class VMInterfaceFilterSet(BaseFilterSet, StatusModelFilterSetMixin, CustomField
         to_field_name="name",
         label="Virtual machine",
     )
-    parent_interface_id = django_filters.ModelMultipleChoiceFilter(
```

Comment:
Since #1981 didn't remove `parent_interface_id` and `bridge_id` from `InterfaceFilterSet`, we should either leave them in here for now to be self-consistent, or remove them from there as well as a part of this PR.